### DISCUSSION TITLE
not validate email on login

### DISCUSSION THREAD

**Discussion by mateusgalasso** [laravel/nova-issues#4652] [Q&A] [answered]
Could you remove email validation in vue?
`type="email"`
I need that the login can be done in 3 different ways.

**Comment by crynobone** [comment, marked as answer]
You should be able to override the Login page using 

```js
Nova.inertia('Nova.Login', require('@/pages/CustomLogin').default)
```

**Reply by mateusgalasso** [reply]
Where?

**Reply by anditsung** [reply]
you need to create custom tool and add this on that tool

**Comment by sanderbaas** [comment]
Has anybody got this working as tools only get loaded after the user has logged in. How can I bypass that?

**Reply by ikaymaz** [reply]
Currently, that's not possible. See: https://github.com/laravel/nova-issues/discussions/4966
